pom.xml

```diff
@@ -70,6 +70,7 @@
   <modules>
     <module>sdk</module>
     <module>examples</module>
+    <module>contrib/hadoop</module>
```

Comment:
Please revert this change. We'd like to have a separate build for the `contrib` module. Perhaps it can be gated by a Maven profile in this file, or a separate aggregator `pom.xml` in contrib/ directory. It's fine to defer all that to a separate pull request, or leave that work to us.